v1.0/Obsolete/Part 3: Guidelines.md

```diff
@@ -55,7 +55,7 @@ The following table gives guidelines for how to comply with the requirements in
 | --- | --- |
 | OOXML_1 | The password can manually be obtained and input to unlock and resave the file without password protection. Obtaining the password may involve searching through records management systems, other registries or asking the creator of the file. If the password cannot be obtained the spreadsheet should not be preserved. |
 | OOXML_2 | File format compliance should be validated using a tool, since the manual inspection of each property is an unfeasible task for any human. |
-| OOXML_3 | Extension should be lowercase.<br><br>Extension “.xlam” is not allowed, since this data are an “macro-enabled workbook add-in” and only contains technical data, that changes or enhances the interface and functionality of the rendering application i.e. Excel. |
+| OOXML_3 | Extension should be lowercase.<br><br>Extension “.xlam” is not allowed, since this data are an “macro-enabled workbook add-in” and only contains technical data, that changes or enhances the interface and functionality of the rendering application i.e. Excel. Extension ".xlsb" is not allowed since this is a binary file format. |
```

Comment:
Be wary of conflating extension and format. They aren't the same thing and are paired by convention. Use MIME types for format and associate the extension with the MIME type in specific requirements.